scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -2041,10 +2177,30 @@ def run( self ):
 
         k8s_cache = self.__get_k8s_cache()
 
+        envars_to_log = [
+            'SCALYR_K8S_NODE_NAME',
+            'SCALYR_K8S_POD_NAME',
+            'SCALYR_K8S_POD_NAMESPACE',
+            'SCALYR_K8S_POD_UID',
+            'SCALYR_K8S_KUBELET_HOST_IP',
+            'SCALYR_K8S_CLUSTER_NAME',
+            'SCALYR_K8S_DISABLE_API_SERVER',
```

Comment:
Also this one should be removed since the `k8s_disable_api_server` feature isn't included in this branch.